docs/android-setup.md

```diff
@@ -0,0 +1,93 @@
+# Installation
+
+In some cases, simply doing `react-native link` will correctly set up the tracker on Android.
+However, if you encounter any issues (either with the command itself or at compile-time/run-time),
+you should revert any changes made by `react-native link` and try the following steps, instead.
+
+Add the following to your `settings.gradle`:
+
+  ```groovy
+  include ':react-native-heap'
+  project(':react-native-heap').projectDir =
+      new File(rootProject.projectDir, '../node_modules/@heap/react-native-heap/android')
+  ```
+
+Then add the following to `app/build.gradle`:
+
+  ```groovy
+  implementation project(':react-native-heap')
+  ```
+
+Import the package and add it to your app's MainApplication:
+
+  ```java
+  @Override
+  import com.heapanalytics.reactnative.RNHeapLibraryPackage;
+  ...
+  protected List<ReactPackage> getPackages() {
+    return Arrays.<ReactPackage>asList(
+      new MainReactPackage(),
+      new RNHeapLibraryPackage()
+    );
+  }
+  ```
+
+If you are seeing runtime warnings at startup mentioning `Heap: Could not find BuildConfig`, add the
+following line to the resources section of `res/values/strings.xml`:
+
+  ```xml
+  <string name="com.heapanalytics.android.buildConfigPkgName">com.your_package_name</string>
+  ```
+
+and replace `com.your_package_name` with the package name from the manifest tag in `AndroidManifest.xml`.
+
+# Confguration
```

Comment:
A lot of this is shared with iOS.  I'd expect the most common use case for RN apps would be to install Heap for both iOS _and_ Android, not just one or the other, so this should probably be pulled out into a common set of instructions.

Reply:
There's enough platform-specific behavior in this section that managing it all in a single doc was unwieldy. I don't think it's unreasonable to expect users to follow two separate docs to integrate android and iOS, even if there's some repetition in there.